translation.py

```diff
@@ -62,7 +62,8 @@ def __del__(self):
     def translate(self, toTranslate):
         with self.use():
             all_split = splitForTranslation(toTranslate, n_users=self.users)
-            parts = yield [translateNULFlush(part, self) for part in all_split]
+            # parts = yield [translateNULFlush(part, self) for part in all_split]
+            parts = yield [translateNULFlush(toTranslate, self)]
```

Comment:
as I said on IRC, add an optional arg "nosplit=False" here; we only want this non-splitting hack if "nosplit=True"